instagram-app/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,105 @@
+import React, {Component} from 'react';
+import uuid from 'uuid';
+import SearchBar from '../SearchBar/SearchBar'
+import dummyData from '../../dummy-data';
+import PostContainer from './PostContainer';
+import AddComment from './AddComment';
+import '../../App.css';
+class PostsPage extends Component {
+  state = {
+    data : [],
+    message: '',
+    searchUser: ''
+  }
+
+  async componentDidMount() {
+    dummyData.map(data=>data.userLike=false)
```

Comment:
What is this line here for?

Reply:
I'm adding a userLike boolean value so that we can validate and ensure a user doesn't like a post more than once